package.json

```diff
@@ -22,17 +23,17 @@
   "author": "Stefan Arentz <stefan@devbots.xyz>",
   "license": "MIT",
   "bugs": {
-    "url": "https://github.com/devbotsxyz/carthage-bootstrap/issues"
+    "url": "https://github.com/DevCycleHQ/carthage-bootstrap/issues"
   },
-  "homepage": "https://github.com/devbotsxyz/carthage-bootstrap#readme",
+  "homepage": "https://github.com/DevCycleHQ/carthage-bootstrap#readme",
   "dependencies": {
-    "@actions/cache": "^1.0.2",
-    "@actions/core": "^1.9.1",
-    "execa": "^4.0.3"
+    "@actions/cache": "^3.2.2",
+    "@actions/core": "^1.10.1",
+    "execa": "^8.0.1"
   },
   "devDependencies": {
-    "@vercel/ncc": "^0.24.1",
-    "eslint": "^7.4.0",
-    "jest": "^26.4.2"
+    "@vercel/ncc": "^0.38.1",
+    "eslint": "^8.52.0",
+    "jest": "^29.7.0"
```

Comment:
This should resolve a lot of the issues we have on dependabot.

Reply:
yeah, some big bumps but given how out of date the repo was it makes sense.